signup.css

```diff
@@ -0,0 +1,131 @@
+.signup{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    background-color: var(--light-blue);
+    border-radius: 8px;
+    width: 90%;
+    max-width: 93.7rem;
+    margin: 0rem auto;
+    padding: 5rem 1rem;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+    gap: 1rem;
+}
+.wrapper{
+    position: relative;
+}
+.blob{
+    position: absolute;
+    width: 30%;
+    z-index: -1;
+}
+.blob1{
+    top: -10px;
+    left: -100px;
+}
+.blob2{
+    bottom: 100px;
+    right: -100px;
+}
+.authentication-image-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column: 1 / 2;
+    z-index: 2;
+    width: 100%; 
+}
+.authentication-image-container img{
+    width: 100%;
+    max-width: 40rem;
+}
+.authentication-form-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column: 2 / 3;
+    z-index: 2;
+}
+.authentication-form{
+    background-color: white;
+    max-width: 31.2rem;
+    width: 100%;
+    border-radius: 6px;
+    padding: 1rem 0.5rem;
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+.authentication-form-heading{
+    text-align: center;
+    margin: 0rem 1rem 0.3rem 1rem;
```

Comment:
0 is just enough rather than using 0rem

Reply:
Actually i like to keep things consistent that's why i use them